pkg/storage/tsdb/index/index.go

```diff
@@ -1865,12 +1869,16 @@ func (dec *Decoder) Series(b []byte, lbls *labels.Labels, chks *[]ChunkMeta) err
 
 	t0 := d.Varint64()
 	maxt := int64(d.Uvarint64()) + t0
+	nBytes := d.Uvarint64()
+	entries := uint32(d.Uvarint64())
```

Comment:
Uvarint32 doesn't exists ?

Reply:
No, it doesn't. It uses 1-10 bytes depending on what is needed to load into a uint64. Since we only encode uint32's, we can safely wrap this with uint32 and it has the added benefit of never needing 10 bytes, but rather 5 max.